Quarkus main branch build on Windows fails with internal Develocity error

### Describe the bug

I build Quarkus main branch in my AWS Windows instance. Lately build started to fail over internal Develocity error. Not sure how I can disable it, I don't want leverage Develocity.

### Expected behavior

Build succeeds.

### Actual behavior

Build fails with:

```
[INFO] -----------< io.quarkus:quarkus-kubernetes-service-binding >------------
[INFO] Building Quarkus - Kubernetes Service Binding - Runtime 999-SNAPSHOT [221/1004]
[INFO]   from extensions\kubernetes-service-binding\runtime\pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- clean:3.2.0:clean (clean-cache-dirs) @ quarkus-kubernetes-service-binding ---
[INFO] 
[INFO] --- clean:3.2.0:clean (default-clean) @ quarkus-kubernetes-service-binding ---
[INFO] 
[INFO] --- buildnumber:3.0.0:create (get-scm-revision) @ quarkus-kubernetes-service-binding ---
[INFO] 
[INFO] --- formatter:2.24.1:format (default) @ quarkus-kubernetes-service-binding ---
[INFO] Processed 9 files in 17ms (Formatted: 0, Skipped: 0, Unchanged: 9, Failed: 0, Readonly: 0)
[INFO] 
[INFO] --- impsort:1.11.0:sort (sort-imports) @ quarkus-kubernetes-service-binding ---
[INFO] Processed 9 files in 00:00.013 (Already Sorted: 9, Needed Sorting: 0)
[INFO] 
[INFO] --- resources:3.3.1:resources (default-resources) @ quarkus-kubernetes-service-binding ---
[INFO] Copying 1 resource from src\main\resources to target\classes
[INFO] 
[INFO] --- quarkus-extension:999-SNAPSHOT:extension-descriptor (generate-extension-descriptor) @ quarkus-kubernetes-service-binding ---
[INFO] 
[INFO] --- compiler:3.13.0:compile (default-compile) @ quarkus-kubernetes-service-binding ---
[INFO] Recompiling the module because of changed dependency.
[INFO] Compiling 7 source files with javac [debug deprecation parameters release 17] to target\classes
[INFO] 
[INFO] --- resources:3.3.1:testResources (default-testResources) @ quarkus-kubernetes-service-binding ---
[INFO] Copying 11 resources from src\test\resources to target\test-classes
[INFO] 
[INFO] --- compiler:3.13.0:testCompile (default-testCompile) @ quarkus-kubernetes-service-binding ---
[ERROR] Internal error in Develocity Maven extension: goal execution
com.gradle.maven.common.logging.DevelocityException: Internal error in Develocity Maven extension: goal execution
    at com.gradle.maven.common.logging.DevelocityException.a (SourceFile:31)
    at com.gradle.maven.cache.extension.d.k.a (SourceFile:104)
    at com.gradle.maven.cache.extension.d.k.a (SourceFile:88)
    at com.gradle.maven.cache.extension.h.b.lambda$createProxy$0 (SourceFile:79)
    at jdk.proxy34.$Proxy144.execute (Unknown Source)
    at com.gradle.maven.scan.extension.internal.d.a.executeMojo (SourceFile:116)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
Caused by: java.lang.RuntimeException: Failed to load cache entry for org.apache.maven.plugins:maven-compiler-plugin:3.13.0:testCompile (default-testCompile) @ io.quarkus:quarkus-kubernetes-service-binding:jar:999-SNAPSHOT
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:87)
    at com.gradle.maven.cache.extension.d.a.b (SourceFile:80)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:50)
    at com.gradle.maven.cache.extension.d.e.a (SourceFile:27)
    at com.gradle.maven.cache.extension.d.l.a (SourceFile:17)
    at com.gradle.maven.cache.extension.d.d.a (SourceFile:47)
    at com.gradle.maven.cache.extension.d.b.a (SourceFile:26)
    at com.gradle.maven.cache.extension.d.f$1.run (SourceFile:35)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:29)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:26)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:66)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:166)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.run (SourceFile:47)
    at com.gradle.maven.cache.extension.d.f.a (SourceFile:31)
    at com.gradle.maven.cache.extension.d.k.a (SourceFile:80)
    at com.gradle.maven.cache.extension.h.b.lambda$createProxy$0 (SourceFile:79)
    at jdk.proxy34.$Proxy144.execute (Unknown Source)
    at com.gradle.maven.scan.extension.internal.d.a.executeMojo (SourceFile:116)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
Caused by: org.gradle.caching.internal.controller.BuildCacheOperationException: Could not load from remote cache: Cannot create directory 'C:\Users\hudson\quarkus\extensions\kubernetes-service-binding\runtime\target\test-classes\k8s\test-k8s\.hidden-symlink-dir'.
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.lambda$loadRemoteAndStoreResultLocally$2 (SourceFile:140)
    at org.gradle.caching.local.internal.DefaultBuildCacheTempFileStore.withTempFile (SourceFile:37)
    at org.gradle.caching.local.internal.DirectoryBuildCache.lambda$withTempFile$5 (SourceFile:175)
    at org.gradle.cache.internal.DefaultCacheCoordinator.lambda$toSupplier$2 (SourceFile:228)
    at org.gradle.cache.internal.LockOnDemandCrossProcessCacheAccess.withFileLock (SourceFile:90)
    at org.gradle.cache.internal.DefaultCacheCoordinator.withFileLock (SourceFile:218)
    at org.gradle.cache.internal.DefaultPersistentDirectoryStore.withFileLock (SourceFile:154)
    at org.gradle.cache.internal.DefaultCacheFactory$ReferenceTrackingCache.withFileLock (SourceFile:211)
    at org.gradle.caching.local.internal.DirectoryBuildCache.withTempFile (SourceFile:175)
    at org.gradle.caching.local.internal.DirectoryBuildCacheService.withTempFile (SourceFile:65)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.loadRemoteAndStoreResultLocally (SourceFile:135)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.load (SourceFile:119)
    at com.gradle.maven.cache.extension.b.c.a (SourceFile:32)
    at com.gradle.maven.cache.extension.b.f$a.a (SourceFile:292)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:85)
    at com.gradle.maven.cache.extension.d.a.b (SourceFile:80)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:50)
    at com.gradle.maven.cache.extension.d.e.a (SourceFile:27)
    at com.gradle.maven.cache.extension.d.l.a (SourceFile:17)
    at com.gradle.maven.cache.extension.d.d.a (SourceFile:47)
    at com.gradle.maven.cache.extension.d.b.a (SourceFile:26)
    at com.gradle.maven.cache.extension.d.f$1.run (SourceFile:35)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:29)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:26)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:66)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:166)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.run (SourceFile:47)
    at com.gradle.maven.cache.extension.d.f.a (SourceFile:31)
    at com.gradle.maven.cache.extension.d.k.a (SourceFile:80)
    at com.gradle.maven.cache.extension.h.b.lambda$createProxy$0 (SourceFile:79)
    at jdk.proxy34.$Proxy144.execute (Unknown Source)
    at com.gradle.maven.scan.extension.internal.d.a.executeMojo (SourceFile:116)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
Caused by: org.gradle.internal.operations.BuildOperationInvocationException: Cannot create directory 'C:\Users\hudson\quarkus\extensions\kubernetes-service-binding\runtime\target\test-classes\k8s\test-k8s\.hidden-symlink-dir'.
    at org.gradle.internal.operations.DefaultBuildOperationRunner.throwAsBuildOperationInvocationException (SourceFile:197)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.access$100 (SourceFile:24)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:75)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:166)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.call (SourceFile:53)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController$PackOperationExecutor.unpack (SourceFile:189)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.lambda$loadRemoteAndStoreResultLocally$1 (SourceFile:138)
    at org.gradle.caching.internal.controller.service.BaseRemoteBuildCacheServiceHandle.maybeUnpack (SourceFile:118)
    at org.gradle.caching.internal.controller.service.BaseRemoteBuildCacheServiceHandle.maybeLoad (SourceFile:105)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.lambda$loadRemoteAndStoreResultLocally$2 (SourceFile:138)
    at org.gradle.caching.local.internal.DefaultBuildCacheTempFileStore.withTempFile (SourceFile:37)
    at org.gradle.caching.local.internal.DirectoryBuildCache.lambda$withTempFile$5 (SourceFile:175)
    at org.gradle.cache.internal.DefaultCacheCoordinator.lambda$toSupplier$2 (SourceFile:228)
    at org.gradle.cache.internal.LockOnDemandCrossProcessCacheAccess.withFileLock (SourceFile:90)
    at org.gradle.cache.internal.DefaultCacheCoordinator.withFileLock (SourceFile:218)
    at org.gradle.cache.internal.DefaultPersistentDirectoryStore.withFileLock (SourceFile:154)
    at org.gradle.cache.internal.DefaultCacheFactory$ReferenceTrackingCache.withFileLock (SourceFile:211)
    at org.gradle.caching.local.internal.DirectoryBuildCache.withTempFile (SourceFile:175)
    at org.gradle.caching.local.internal.DirectoryBuildCacheService.withTempFile (SourceFile:65)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.loadRemoteAndStoreResultLocally (SourceFile:135)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.load (SourceFile:119)
    at com.gradle.maven.cache.extension.b.c.a (SourceFile:32)
    at com.gradle.maven.cache.extension.b.f$a.a (SourceFile:292)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:85)
    at com.gradle.maven.cache.extension.d.a.b (SourceFile:80)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:50)
    at com.gradle.maven.cache.extension.d.e.a (SourceFile:27)
    at com.gradle.maven.cache.extension.d.l.a (SourceFile:17)
    at com.gradle.maven.cache.extension.d.d.a (SourceFile:47)
    at com.gradle.maven.cache.extension.d.b.a (SourceFile:26)
    at com.gradle.maven.cache.extension.d.f$1.run (SourceFile:35)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:29)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:26)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:66)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:166)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.run (SourceFile:47)
    at com.gradle.maven.cache.extension.d.f.a (SourceFile:31)
    at com.gradle.maven.cache.extension.d.k.a (SourceFile:80)
    at com.gradle.maven.cache.extension.h.b.lambda$createProxy$0 (SourceFile:79)
    at jdk.proxy34.$Proxy144.execute (Unknown Source)
    at com.gradle.maven.scan.extension.internal.d.a.executeMojo (SourceFile:116)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
Caused by: java.io.IOException: Cannot create directory 'C:\Users\hudson\quarkus\extensions\kubernetes-service-binding\runtime\target\test-classes\k8s\test-k8s\.hidden-symlink-dir'.
    at com.gradle.maven.extension.internal.dep.org.apache.commons.io.FileUtils.mkdirs (SourceFile:2289)
    at com.gradle.maven.extension.internal.dep.org.apache.commons.io.FileUtils.forceMkdir (SourceFile:1376)
    at org.gradle.caching.internal.packaging.impl.TarBuildCacheEntryPacker.unpackDirectoryTree (SourceFile:298)
    at org.gradle.caching.internal.packaging.impl.TarBuildCacheEntryPacker.unpackTree (SourceFile:266)
    at org.gradle.caching.internal.packaging.impl.TarBuildCacheEntryPacker.unpack (SourceFile:210)
    at org.gradle.caching.internal.packaging.impl.TarBuildCacheEntryPacker.unpack (SourceFile:172)
    at org.gradle.caching.internal.packaging.impl.GZipBuildCacheEntryPacker.unpack (SourceFile:49)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController$PackOperationExecutor.doUnpack (SourceFile:210)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController$PackOperationExecutor.access$000 (SourceFile:174)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController$PackOperationExecutor$1.call (SourceFile:194)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController$PackOperationExecutor$1.call (SourceFile:189)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute (SourceFile:209)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute (SourceFile:204)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:66)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:166)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.call (SourceFile:53)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController$PackOperationExecutor.unpack (SourceFile:189)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.lambda$loadRemoteAndStoreResultLocally$1 (SourceFile:138)
    at org.gradle.caching.internal.controller.service.BaseRemoteBuildCacheServiceHandle.maybeUnpack (SourceFile:118)
    at org.gradle.caching.internal.controller.service.BaseRemoteBuildCacheServiceHandle.maybeLoad (SourceFile:105)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.lambda$loadRemoteAndStoreResultLocally$2 (SourceFile:138)
    at org.gradle.caching.local.internal.DefaultBuildCacheTempFileStore.withTempFile (SourceFile:37)
    at org.gradle.caching.local.internal.DirectoryBuildCache.lambda$withTempFile$5 (SourceFile:175)
    at org.gradle.cache.internal.DefaultCacheCoordinator.lambda$toSupplier$2 (SourceFile:228)
    at org.gradle.cache.internal.LockOnDemandCrossProcessCacheAccess.withFileLock (SourceFile:90)
    at org.gradle.cache.internal.DefaultCacheCoordinator.withFileLock (SourceFile:218)
    at org.gradle.cache.internal.DefaultPersistentDirectoryStore.withFileLock (SourceFile:154)
    at org.gradle.cache.internal.DefaultCacheFactory$ReferenceTrackingCache.withFileLock (SourceFile:211)
    at org.gradle.caching.local.internal.DirectoryBuildCache.withTempFile (SourceFile:175)
    at org.gradle.caching.local.internal.DirectoryBuildCacheService.withTempFile (SourceFile:65)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.loadRemoteAndStoreResultLocally (SourceFile:135)
    at org.gradle.caching.internal.controller.DefaultBuildCacheController.load (SourceFile:119)
    at com.gradle.maven.cache.extension.b.c.a (SourceFile:32)
    at com.gradle.maven.cache.extension.b.f$a.a (SourceFile:292)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:85)
    at com.gradle.maven.cache.extension.d.a.b (SourceFile:80)
    at com.gradle.maven.cache.extension.d.a.a (SourceFile:50)
    at com.gradle.maven.cache.extension.d.e.a (SourceFile:27)
    at com.gradle.maven.cache.extension.d.l.a (SourceFile:17)
    at com.gradle.maven.cache.extension.d.d.a (SourceFile:47)
    at com.gradle.maven.cache.extension.d.b.a (SourceFile:26)
    at com.gradle.maven.cache.extension.d.f$1.run (SourceFile:35)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:29)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute (SourceFile:26)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:66)
    at org.gradle.internal.operations.DefaultBuildOperationRunner$2.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:166)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.execute (SourceFile:59)
    at org.gradle.internal.operations.DefaultBuildOperationRunner.run (SourceFile:47)
    at com.gradle.maven.cache.extension.d.f.a (SourceFile:31)
    at com.gradle.maven.cache.extension.d.k.a (SourceFile:80)
    at com.gradle.maven.cache.extension.h.b.lambda$createProxy$0 (SourceFile:79)
    at jdk.proxy34.$Proxy144.execute (Unknown Source)
    at com.gradle.maven.scan.extension.internal.d.a.executeMojo (SourceFile:116)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute2 (MojoExecutor.java:328)
    at org.apache.maven.lifecycle.internal.MojoExecutor.doExecute (MojoExecutor.java:316)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:212)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:174)
    at org.apache.maven.lifecycle.internal.MojoExecutor.access$000 (MojoExecutor.java:75)
    at org.apache.maven.lifecycle.internal.MojoExecutor$1.run (MojoExecutor.java:162)
    at org.apache.maven.plugin.DefaultMojosExecutionStrategy.execute (DefaultMojosExecutionStrategy.java:39)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:159)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:105)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:73)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:53)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:118)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:261)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:173)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:101)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:906)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:283)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:206)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:255)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:201)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:361)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:314)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.apache.maven.wrapper.BootstrapMainStarter.start (BootstrapMainStarter.java:52)
    at org.apache.maven.wrapper.WrapperExecutor.execute (WrapperExecutor.java:161)
    at org.apache.maven.wrapper.MavenWrapperMain.main (MavenWrapperMain.java:73)
```


### How to Reproduce?

Steps to reproduce the issue:

```
git clone https://github.com/quarkusio/quarkus.git
cd quarkus
cmd /c mvnw.cmd -B -s .github/mvn-settings.xml clean install -Dquickly -Dno-test-modules -Prelocations 
```

### Output of `uname -a` or `ver`

Windows Server 2022

### Output of `java -version`

Temurin 17

### Quarkus version or git rev

999-SNAPSHOT

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.9.9

### Additional information

I'd appreciate a workaround as this is blocking me. @gsmet perhaps my build command is not valid anymore?

Ok.... What I can say for now:
 Some time ago, build was successful.
I've just did `git fetch upstream` and checked onto main, and this error started to pop up on `./mvnw -Dquickly` execution.
https://pastebin.com/LTWUJjff

I have Java 17 installed on the PATH and JAVA_HOME.
Windows 11 with fresh updates.
Also, I've installed Maven 3.9.9 locally to try to build not via wrapper. But got same error. 
Never tried to use 3.7.7 to see if problem in Maven version.

Also, as @jprinet asked, I have removed the `.mvn/extensions.xml` file, but got another error:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.5.0:test (default-test) on project quarkus-development-mode-spi: Unable to parse configuration of mojo org.apache.maven.plugins:maven-surefire-plugin:3.5.0:test for parameter properties: Missing name in properties
```

Unfortunately, the error has returned.

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:testCompile (default-testCompile) on project quarkus-kubernetes-service-binding: Execution default-testCompile of goal org.apache.maven.plugins:maven-compiler-plugin:3.13.0:testCompile failed: Internal error in Develocity Maven extension: goal execution: Failed to load cache entry for org.apache.maven.plugins:maven-compiler-plugin:3.13.0:testCompile (default-testCompile) @ io.quarkus:quarkus-kubernetes-service-binding:jar:999-SNAPSHOT: Could not load from remote cache: Cannot create directory 'C:\Users\asus\repo\quarkus\extensions\kubernetes-service-binding\runtime\target\test-classes\k8s\test-k8s\.hidden-symlink-dir'. -> [Help 1]
```
How that can be truth ?

So, local cache working, but remote nope ?

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -252,6 +252,7 @@ If you have not done so on this machine, you need to:
   * Windows:
     * enable longpaths: `git config --global core.longpaths true`
     * avoid CRLF breaks: `git config --global core.autocrlf false`
+    * enable symlinks: `git config --global core.symlinks true`
 * Install Java SDK 17+ (OpenJDK recommended)
 * Install [GraalVM](https://quarkus.io/guides/building-native-image)
 * Install platform C developer tools:

